_data/shakeelsamsu/shakeelsamsu.md

```diff
@@ -0,0 +1,7 @@
+---
+name: shakeelsamsu
+institution: PSHS -> UT Austin
+profile_pic: shakeelsamsu.png
+quote: "Always seek answers to questions considered unanswerable"
```

Comment:
Hey @shakeelsamsu , could you avoid using ( " ) in the quote, just to keep the formatting intact ?

Reply:
Will do. I added them because the example, `MonaTheOctocat.md`, included quotation marks.